lib/pipeline/pipeline.js

```diff
@@ -85,11 +85,14 @@ class Pipeline {
       },
       function runWebpack(next) {
         self.logger.info(__(`running webpack with '${self.webpackConfigName}' config...`));
-        Object.keys(self.assetFiles)
-          .filter(key => key.match(/\.js?$/))
-          .forEach(key => {
-            self.logger.info(__("writing file") + " " + (utils.joinPath(self.buildDir, key)).bold.dim);
-          });
+        const assets = Object.keys(self.assetFiles)
+          .filter(key => key.match(/\.js?$/));
+        assets.forEach(key => {
+          self.logger.info(__("writing file") + " " + (utils.joinPath(self.buildDir, key)).bold.dim);
+        });
+        if (!assets || !assets.length) {
```

Comment:
Shouldn't this be before the `forEach` loop? Sounds like we can potentially avoid that call

Reply:
Yeah I guess. Though, `assets` just becomes `[]`  so forEach does nothing on it.
I'll still do it since it makes sense